src/placeos-driver/queue.cr

```diff
@@ -44,7 +43,7 @@ class PlaceOS::Driver::Queue
   # removes all jobs currently in the queue
   def clear(abort_current = false)
     old_queue = @queue
```

Comment:
Is there a chance of async access to the driver queue? If so, I think there should be synchronisation around access to `@queue`